<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="seleniumcn.cn.demo" duration-ms="106788" started-at="2015-04-15T14:06:48Z" finished-at="2015-04-15T14:08:34Z">
    <groups>
    </groups>
    <test name="test_seleniumcn" duration-ms="106788" started-at="2015-04-15T14:06:48Z" finished-at="2015-04-15T14:08:34Z">
      <class name="test.HelloworldWebdriverTestNG">
        <test-method status="PASS" signature="testOne()[pri:0, instance:test.HelloworldWebdriverTestNG@13b6d03]" name="testOne" duration-ms="48611" started-at="2015-04-15T22:06:48Z" finished-at="2015-04-15T22:07:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOne -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.HelloworldWebdriverTestNG@13b6d03]" name="tearDown" is-config="true" duration-ms="9701" started-at="2015-04-15T22:07:36Z" finished-at="2015-04-15T22:07:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.HelloworldWebdriverTestNG -->
      <class name="test.HelloworldWebdriverTestNG_Fail">
        <test-method status="FAIL" signature="testOne_Fail()[pri:0, instance:test.HelloworldWebdriverTestNG_Fail@445b84c0]" name="testOne_Fail" duration-ms="40536" started-at="2015-04-15T22:07:46Z" finished-at="2015-04-15T22:08:26Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Selenium 中文论坛, Webdriver 中文论坛, Selenium2, 软件测试,web自动化测试_Fail] but found [selenium,webdriver,Selenium 中文论坛,Webdriver 中文论坛,Selenium2,软件测试,web自动化测试 - Powered by phpwind]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Selenium 中文论坛, Webdriver 中文论坛, Selenium2, 软件测试,web自动化测试_Fail] but found [selenium,webdriver,Selenium 中文论坛,Webdriver 中文论坛,Selenium2,软件测试,web自动化测试 - Powered by phpwind]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at test.HelloworldWebdriverTestNG_Fail.testOne_Fail(HelloworldWebdriverTestNG_Fail.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.TestNG.privateMain(TestNG.java:1364)
	at org.testng.TestNG.main(TestNG.java:1333)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOne_Fail -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.HelloworldWebdriverTestNG_Fail@445b84c0]" name="tearDown" is-config="true" duration-ms="7920" started-at="2015-04-15T22:08:26Z" finished-at="2015-04-15T22:08:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.HelloworldWebdriverTestNG_Fail -->
    </test> <!-- test_seleniumcn -->
  </suite> <!-- seleniumcn.cn.demo -->
</testng-results>
